scripts/workers/submission_worker.py

```diff
@@ -186,6 +191,7 @@ def extract_challenge_data(challenge, phases):
     evaluation_script_url = return_file_url_per_environment(evaluation_script_url)
     # create challenge directory as package
     create_dir_as_python_package(challenge_data_directory)
+
```

Comment:
Can we please remove this line from here? It has no significance related to Python3 migration actually.